.github/workflows/bare.yaml

```diff
@@ -38,8 +38,8 @@ jobs:
       if: startsWith(matrix.runs-on, 'ubuntu')
       run: sudo apt-get install -y libncurses5-dev
 
-    # GHA's checkout action seems to be misbehaving on ARM64 and consequently
-    # our build script can't get current revision from cloned sources.
+    # Git's version on our ARM64 runner is too old for actions/checkout to
+    # create a local repo. Beware this fails on opening a new PR.
     - name: Clone sources independently
       run: |
         git init $CHEVAH_REPO
```

Comment:
I think that we should add a final "git log -1 --format='%H'" to show the current HEAD .
This is what actions/checkout does and I forgot to add it.

Reply:
But this would just show something, how it would fix anything?

Reply:
It will not fix...is just something that I found that actions/checkout does and the custom script is not doing.

It is there only to help with debugging.

And I think that an `ls -al .` can also help  with debugging.

I will experiment with it on chevah/server and will let you know how it goes.

Reply:
Thanks! Is this the reason you have not approved this? Or what is preventing it?

Reply:
I've added your suggested command. Thanks!